src/main/resources/META-INF/plugin.xml

```diff
@@ -3,7 +3,13 @@
     <name>Zalando OpenAPI Editor</name>
     <vendor email="sebastian.monte@zalando.de" url="https://tech.zalando.com/">Zalando SE</vendor>
 
+    <!-- Module dependencies -->
+    <depends>com.intellij.modules.platform</depends>
     <depends>com.intellij.modules.lang</depends>
+    <depends>com.intellij.modules.json</depends>  <!-- Bundled plugin starting from 2024.3 -->
```

Comment:
So if the previous versions of IntellIj don't have this, will it still work with older versions?

Reply:
Thanks for explanation. But how would we then find out? Just merging this PR and see what happens?

Reply:
Generally yes. We can do a release to the beta channel.